### PR TITLE
LFX Platform Migrating to PSI Bridge remote desktop

### PR DESCRIPTION
**Pull request by seifrajhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,18 +36,23 @@ Offical exam objectives you review and understand in order to pass the test.
 
   *Linux Foundation offer several discounts around the year e.g. CyberMonday, Kubecon attendees among other special holidays/events*
 
-### URLs allowed in the extra single tab
```
Comment (walidshaari) [inline README.md:39]: Why did you remove that section?

Comment (walidshaari) [reply README.md:39]: is it not applicable on the remote browser?

**Review by walidshaari — COMMENTED** [review]
Why did you remove the section of Allowed URLs? is it not applicable any more due to the remote desktop? could you please explain?

**Comment by seifrajhi** [comment]
Yes, I have passed the exam.
The exam environment will now take place remotely on a Linux desktop.
It is a remote desktop on PSI Secure Browser, configured with the tools and software needed to complete the tasks.

Maybe, you can keep the URLs to prepare for the exams, but bookmarks are not accessible anymore during exam.

References: 
https://training.linuxfoundation.org/bridge-migration-2021/
https://itnext.io/cks-cka-ckad-changed-terminal-to-remote-desktop-157a26c1d5e
https://twitter.com/_killer_shell/status/1541743761180565505?lang=en

**Comment by walidshaari** [comment]
So you need to update your pull request not to remove unnecessary changes. in general pull requests that are minimal are easy to approve

**Comment by seifrajhi** [comment]
Ok
I have updated the pull request as demanded
